datasets/cord-19/README.md

```diff
@@ -0,0 +1,172 @@
+---
+annotations_creators:
+- no-annotation
+language_creators:
+- expert-generated
+languages:
+- en
+licenses:
+- 'other-multiple licenses: ''cc0'', ''hybrid-oa'', ''els-covid'', ''no-cc'', ''cc-by-nc-sa'',
+  ''cc-by'', ''gold-oa'', ''biorxiv'', ''green-oa'', ''bronze-oa'', ''cc-by-nc'',
+  ''medrxiv'', ''cc-by-nd'', ''arxiv'', ''unk'', ''cc-by-sa'', ''cc-by-nc-nd'''
```

Comment:
Can you list the licenses like this
```
licenses:
- license-name-1
- license-name-2
- other-license-name-3-that-is-not-available-in-the-tagging-app
```

?

You can use the tagging app to help you do that.
More info here : https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#tag-the-dataset-and-write-the-dataset-card